src/components/question/answers-table.cjsx

```diff
@@ -54,14 +55,29 @@ AnswersTable = React.createClass
       changeEvent.preventDefault()
       @props.onChangeAttempt?(answer)
 
+  hasCorrectAnswer: ->
+    {correct_answer_id} = @props
+
+    !! correct_answer_id
+
+  shouldInstructionsShow: ->
+    {type, model} = @props
+    hasCorrectAnswer = @hasCorrectAnswer()
+
+    model.formats.length > 1 and not (hasCorrectAnswer or type in ['teacher-preview', 'teacher-review'])
+
   render: ->
-    {model, type, answered_count, choicesEnabled, correct_answer_id, answer_id, feedback_html, show_all_feedback, keySet} = @props
+    {
+      model, type, answered_count, choicesEnabled, correct_answer_id,
+      answer_id, feedback_html, show_all_feedback, keySet, project
+    } = @props
+
     {answers, id} = model
     return null unless answers?.length > 0
 
     chosenAnswer = [answer_id, @state.answer_id]
     checkedAnswerIndex = null
-    hasCorrectAnswer = !! correct_answer_id
+    hasCorrectAnswer = @hasCorrectAnswer()
 
```

Comment:
I don't see where the `hasCorrectAnswer` var is being used.  I was going to suggest not storing a variable and just call `@hasCorrectAnswer()` where needed, but I don't see that spot 😄